src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/AssemblyReferenceResolverTests.cs

```diff
@@ -39,16 +39,16 @@ public void ShouldAnalyzeProjectDependencies()
             var result = target.ResolveReferences(Path.GetDirectoryName(project), Path.GetFileName(project), "ExampleProject").ToList();
 
             processExecutorMock.Verify(x => x.Start(
-                Path.GetDirectoryName(project), 
-                "dotnet", "msbuild ExampleProject.Test.csproj /nologo /t:PrintReferences", 
+                Path.GetDirectoryName(project),
+                "dotnet", "msbuild ExampleProject.Test.csproj /nologo /t:PrintReferences",
                 null,
-                It.IsAny<int>()), 
+                It.IsAny<int>()),
                 Times.Once);
             foundReferences.ShouldBeSubsetOf(new List<string>()
             {
-                Path.Combine($@"C:{Path.DirectorySeparatorChar}Repos", "ExampleProject", "ExampleProject", "bin", "Debug", "netcoreapp2.0", "ExampleProject.Contracts.dll"),
-                Path.Combine($@"C:{Path.DirectorySeparatorChar}Program Files", "dotnet", "sdk", "NuGetFallbackFolder", "microsoft.netcore.app", "2.0.0", "ref", "netcoreapp2.0", "Microsoft.CSharp.dll"),
-                Path.Combine($@"C:{Path.DirectorySeparatorChar}Program Files", "dotnet", "sdk", "NuGetFallbackFolder", "microsoft.aspnetcore.mvc.formatters.json", "2.0.2", "lib", "netstandard2.0", "Microsoft.AspNetCore.Mvc.Formatters.Json.dll")
+                Path.Combine("C:", "Repos", "ExampleProject", "ExampleProject", "bin", "Debug", "netcoreapp2.0", "ExampleProject.Contracts.dll"),
```

Comment:
This unit test is failing on windows due to this Path.Combine. The expected values are C:Repos but the actual values are C:\Repos

Reply:
Interesting.  I'll admit I didn't try it on Windows, and the GitHub outage meant I couldn't see the failure right until today.

Is C:Repos correct?  If so, what does that represent?  Also, what happens to {Path.DirectorySeparatorChar} in the old code that causes it to disappear?

Reply:
I've tried this now in a Windows VM, and all tests pass for me in Visual Studio.  I see that this test fails in the automated test runs on GitHub, but I'm not sure what would be causing the difference.

Reply:
I just tried it on my local system and the test fails there as well. For some reason on windows the DirectorySeparatorChar is not automatically added to the drive letter even though according to the docs it should be. https://docs.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=netframework-4.7.2#System_IO_Path_Combine_System_String_System_String_

Reply:
Windows 7 seems to behave like Linux.  I don't have another version available to try things on, so I'll just revert that part.